snappy/security.go

```diff
@@ -319,6 +320,13 @@ func findWhitespacePrefix(t string, s string) string {
 	return subs[1]
 }
 
+// XXX: what to use really?
```

Comment:
This is all going away imminently. Just tuck it away in the cheapest possible way, ideally grouping with related logic that will all being dropped with it. @zyga should be able to provide more details if you need it.